### ISSUE TITLE
Win16 Server and Win10 Pro?

### ISSUE THREAD

**Issue by ryanpek** [solusio/solus-cloud-images#5]
What is the ETA for 2016 and win10 pro json script files?

Does #cloud-config user data have the function to set static IPs on windows via cloud-init?

**Comment by pabla** [comment]
ср, 17 июня 2020 г., 4:31 orbwalk <notifications@github.com>:

**Comment by oneumyvakin** [comment]
@ryanpek 

> Does #cloud-config user data have the function to set static IPs on windows via cloud-init?

Yes, it does. Currently there issue that if IP and Gateway in different subnets, gateway maybe not set. We will going to fix this issue in SolusIO via special scripts like we doing it for some Linux OSes. 

> What is the ETA for 2016 and win10 pro json script files?

Currently there is no ETA for 2016.

Could you please share your case with Win10?
What is final purpose to create virtual servers with desktop OS?

**Comment by ryanpek** [comment]
> Could you please share your case with Win10?
> What is final purpose to create virtual servers with desktop OS?

We have lots of clients who use win10 to have a virt windows desktop who do not want windows server.

**Comment by ryanpek** [comment]
How can I use cloudbase-init to set windows 2019 administrator passwords from what's set on the 'create vm' page in solus.io?

**Comment by oneumyvakin** [comment]
@ryanpek we have add Windows 2019 images to SolusIO so you can try use them. Also we have update scripts in repo to resolve authentication and network issues with Windows 2019.

We will think about adding Windows 10 Pro scripts in repo and maybe images in SolusIO.

**Comment by debugged-tech** [comment]
Are any updates on Windows 10 and Windows 11 being added?
